srcDoc/release_notes.rst

```diff
@@ -187,6 +187,13 @@ instead, *RTI Perftest* would always use the default value set up via code.
 This behavior has been fixed. We also took the oportunity to simplify and clarify
 the XML configurations when disabling positive Acks.
 
+Show message in sumary when -multicast is present but it wont be used (#44)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In previous versions, if the `-multicast` command line parameter was provided but
+the transport does not allow it, any information was given. This fix prints the
+message: `Use Multicast: False  (Multicast is not supported for TCP)`.
+
```

Comment:
We have to re-write it. 

Key ideas:
- Now we have two parameters: ``-multicast``, ``-multicastAddr <address>``
- Both parameters will set multicat.
- Now when multicast is set and but it is not allowed. A message will be printed: `Use Multicast: False  (Multicast is not supported for <Transport_Kind>)`.

Reply:
Done. Thanks